src/FsCheck/Gen.fs

```diff
@@ -313,6 +313,24 @@ module Gen =
     [<CompiledName("ListOf")>]
     let listOfLength n arb = sequence [ for _ in 1..n -> arb ]
 
+    ///Generates a random permutation of the given sequence using the Fisher-Yates shuffle algorithm.
```

Comment:
Apologies,  I missed these comments before I merged. Happy to take further improvements as further PRs though.